.travis.yml

```diff
@@ -10,7 +10,10 @@ matrix:
 before_install:
     - pip install pipenv
     - pipenv install --dev --skip-lock
+    - pip install codespell flake8
 script:
+    - codespell -L paket --skip=./textstat/resources/*/easy_words.txt
```

Comment:
Could we preface this with `pipenv run` to make sure we're using the virtual envs `codespell`?